src/server/scripts/Northrend/FrozenHalls/PitOfSaron/boss_scourgelord_tyrannus.cpp

```diff
@@ -97,6 +97,7 @@ class boss_tyrannus : public CreatureScript
                     if (Vehicle* v = c->GetVehicleKit())
                         v->InstallAllAccessories(false);
                 }
+                pInstance->SetData(DATA_TYRANNUS, NOT_STARTED);
```

Comment:
And this line is useless, it isn't doing anything. State is only being changed from NOT_STARTED when its set to DONE (on death).